docs/checkers/index.md

```diff
@@ -1169,6 +1176,27 @@ lines: 1-3
 ---
 ```
 
+PythonTA allows you to specify all the modules that you wish to allow for a particular file using the
+allowed-import-modules configuration option in the check_all function
+
+```python
+import python_ta
+python_ta.check_all(..., config={'allowed-import-modules': ["random"]})
+```
+
+You can specify any additional modules you want to allow for import using the extra-imports
+configuration option, which you can set in a call to python_ta.check_all or in a configuration file.
```

Comment:
as above, you can omit the part after the comma and just use a colon to introduce the subsequent example